go.mod

```diff
@@ -77,7 +77,7 @@ require (
 	k8s.io/api v0.31.0-alpha.3
 	k8s.io/apimachinery v0.31.0-alpha.3
 	k8s.io/client-go v0.31.0-alpha.3
-	k8s.io/cri-api v0.31.0-alpha.3
+	k8s.io/cri-api v0.31.0-beta.0.0.20240716205706-865479a3e1b3
```

Comment:
```suggestion
	k8s.io/cri-api v0.31.0-beta.0
```

Reply:
@saschagrunert Sorry, unfortunately my k/k PR missed v1.31.0-beta.0😿 So, I need the latest version.

This is a commit in cri-api repo: https://github.com/kubernetes/cri-api/commit/865479a3e1b3625494674397b1abc5f72702ca2e (this is not included in the `0.31.0-beta.0` tag)